src/core/Form/Combobox/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,595 @@
+import React, { Component } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlDiv } from '../../../../reset';
+import { windowAvailable } from '../../../../utils/common';
+import { AutoId } from '../../../../utils/AutoId';
+import { Debounce } from '../../../utils/Debounce/Debounce';
+import { Button } from '../../../Button/Button';
+import { Chip } from '../../../Chip';
+import { Popover } from '../../../Popover/Popover';
+import { FilterInput, FilterInputStatus } from '../../FilterInput/FilterInput';
+import { ComboboxItemList } from '../ComboboxItemList/ComboboxItemList';
+import { ComboboxItem } from '../ComboboxItem/ComboboxItem';
+import { ComboboxEmptyItem } from '../ComboboxEmptyItem/ComboboxEmptyItem';
+import { ChipList } from '../ChipList/ChipList';
+import { baseStyles } from './Combobox.baseStyles';
+
+const baseClassName = 'fi-combobox';
+const comboboxClassNames = {
+  wrapper: `${baseClassName}_wrapper`,
+  open: `${baseClassName}--open`,
+  removeAllButton: `${baseClassName}_removeAllButton`,
+  error: `${baseClassName}--error`,
+};
+
+export interface ComboboxData {
+  /** Unique label that will be shown on combobox item and used on filter */
+  labelText: string;
+  /** Using labelText if chipText is not given */
+  chipText?: string;
+  /** Item selection disabled for the user */
+  disabled?: boolean;
+  /** Unique id to identify the item */
+  uniqueItemId: string;
+}
+
+export interface ComboboxProps<T extends ComboboxData> {
+  /** Combobox container div class name for custom styling. */
+  className?: string;
+  /** Items for the combobox */
+  items: Array<T & ComboboxData>;
+  /**
+   * Unique id
+   * If no id is specified, one will be generated automatically
+   */
+  id?: string;
+  /** Label */
+  labelText: string;
+  /** Event that is sent when item selections change */
+  onItemSelectionsChange?: (selectedItems: Array<T>) => void;
+  /** Show chip list */
+  chipListVisible?: boolean;
+  /** Chip action label */
```

Comment:
I guess this is referring to clicking a chip, if this is not visible, I think name should contain aria somehow.

Reply:
Would `ariaChipActionLabel` be good candidate?

It seems that Chip component itself is missing the aria from prop name too: 
https://github.com/vrk-kpa/suomifi-ui-components/blob/a5d6f9bc378805114682b0adca1d15fbb2cb20f0/src/core/Chip/Chip/Chip.tsx#L36-L37

Reply:
Sounds a bit funny this way, but it would be better in line with what we have elsewhere and would make it distinguishable from other props without reading the API desc.

Reply:
This is now changed to `ariaChipActionLabel`.